lib/suite.js

```diff
@@ -594,11 +594,11 @@ Suite.prototype.serialize = function serialize() {
   return {
     _bail: this._bail,
     $$fullTitle: this.fullTitle(),
-    $$isPending: this.isPending(),
+    $$isPending: !!this.isPending(),
     root: this.root,
     title: this.title,
-    id: this.id,
-    parent: this.parent ? {[MOCHA_ID_PROP_NAME]: this.parent.id} : null
+    parent: this.parent ? this.parent.serialize() : null,
+    [MOCHA_ID_PROP_NAME]: this[MOCHA_ID_PROP_NAME]
```

Comment:
agreed, `id` has no setter, so: `[MOCHA_ID_PROP_NAME]: this.id`

Reply:
Done.